spec/v1/envelope.md

```diff
@@ -19,5 +21,41 @@ standard DSSE fields.
     specifying its schema.
 -   `payload` MUST be a base64-encoded JSON [Statement].
 
+## File naming convention
+
+Envelopes SHOULD use the suffix `.json`.
+
+-   An Envelope containing an attestation about a particular SW supply chain
+    step `<step-name>` SHOULD be named `<step-name>.json`.
+-   If multiple Envelopes are produced for the same step by different
+    [functionaries] uniquely identified by a public key, an Envelope name
+    SHOULD include the hash of the public key `<pubkey-hash>` of the signing
+    functionary: `<step-name>.<pubkey-hash>.json`.
+
+## Storage convention
+
+The media type `application/vnd.in-toto.<predicate>+<sig>` SHOULD
+be used to denote an individual attestation in arbitrary storage systems.
+
+-   The `<predicate>` MUST match the [predicate specification name]
```

Comment:
ITE-9 / documentation here may benefit from adding a field for the predicate name alongside the type field instead of relying on the file names.

Reply:
When we discussed this at the Aug 11 maintainer's meeting, we felt the predicate spec filename was the most succinct and uniform way of deriving a predicate name without having to change the Statement schema or create some ad-hoc list with TypeURI-to-name mappings. Do you have any particular concerns around using the predicate spec filename?

Reply:
Not per se. Note that I'm not saying we should update the Statement schema, but rather the format ITE-9 specifies for the predicate specification. So alongside the type, you'd also have the name. I think this is functionally equivalent to using the file name, just that all the information can be derived from the same place.

Reply:
Thanks for clarifying, yes, I had misunderstood your previous comment. We can certainly update ITE-9 for this. Who would be responsible for making that change? You as ITE-9 sponsor?

Reply:
https://github.com/in-toto/ITE/pull/52

Opened a PR.

Reply:
BTW, I could also make this change proposal, I just wasn't sure about the expectations are for making edits to ITEs.

Reply:
Thank you @adityasaky ! I'll go take a look.